db/migrate/20191109222528_change_resource.rb

```diff
@@ -0,0 +1,13 @@
+class ChangeResource < ActiveRecord::Migration[5.2]
+  def change
+    change_table :resources do |t|
+      t.text  :where
+      t.text  :what_to_expect, default: [], array: true
+      t.text  :what_if_i_disagree, default: [], array: true
+      t.text  :resources, default: [], array: true
+      t.text  :steps, default: [], array: true
```

Comment:
A bunch of new fields that reflect the finalized shape of the resource data (see the sample data from Texas above).